sup3r/preprocessing/batch_handling.py

```diff
@@ -90,6 +112,71 @@ def reduce_features(high_res, output_features_ind=None):
         else:
             return high_res[..., output_features_ind]
 
+    # pylint: disable=W0613
+    @staticmethod
+    def make_output(low_res, high_res,
+                    s_enhance=None, t_enhance=None,
+                    model_mom1=None, output_features_ind=None):
+        """Make custom batch output
+
+        Parameters
+        ----------
+        low_res : np.ndarray
+            4D | 5D array
+            (batch_size, spatial_1, spatial_2, features)
+            (batch_size, spatial_1, spatial_2, temporal, features)
+        high_res : np.ndarray
+            4D | 5D array
+            (batch_size, spatial_1, spatial_2, features)
+            (batch_size, spatial_1, spatial_2, temporal, features)
+        s_enhance : int | None
+            Spatial enhancement factor
+        t_enhance : int | None
+            Temporal enhancement factor
+        model_mom1 : Sup3rCondMom | None
+            Model used to modify the make the batch output
+        output_features_ind : list | np.ndarray | None
+            List/array of feature channel indices that are used for generative
+            output, without any feature indices used only for training.
+        """
+        return high_res
+
+    # pylint: disable=E1130
+    @staticmethod
+    def make_mask(high_res, s_padding=None, t_padding=None):
+        """Make mask for output
```

Comment:
Can you say more about what the mask is used for and why? The doc string here doesn't say much more than the function name ;)